api/v330/docking_event/serializers.py

```diff
@@ -19,7 +19,7 @@ class Meta:
 
 
 class DockingEventDetailedSerializer(serializers.HyperlinkedModelSerializer):
-    flight_vehicle = SpacecraftFlightSerializerForDockingEvent(read_only=True, many=False)
+    flight_vehicle = SpacecraftFlightMiniSerializer(read_only=True, many=False)
```

Comment:
Instead of creating an entirely new serializer - edit the SpacecraftFlightSerializerForDockingEvent.